docs/dev/plugins/reporting.md

```diff
@@ -1,313 +1 @@
-
-# Adding basic authentication: https://mailpit.axllent.org/docs/configuration/passwords/
-# mailpit --ui-auth-file /path/to/password-file
-```
-
-</p>
-</details>
-
-**gencerts.sh**
-<details><summary>Details</summary>
-<p>
-
-```bash
-#!/bin/bash
-
-if [[ $# -ne 1 ]]; then
-fs=$(mktemp -d)
-else
-fs=$1
-shift
-fi
-
-echo Working directory $fs
-cd $fs
-
-if [[ ! -e $fs/cfssl ]]; then
-curl -s -L -o $fs/cfssl https://pkg.cfssl.org/R1.2/cfssl_linux-amd64
-curl -s -L -o $fs/cfssljson https://pkg.cfssl.org/R1.2/cfssljson_linux-amd64
-chmod 755 $fs/cfssl*
-fi
-
-cfssl=$fs/cfssl
-cfssljson=$fs/cfssljson
-
-if [[ ! -e $fs/ca.pem ]]; then
-
-cat << EOF | $cfssl gencert -initca - | $cfssljson -bare ca -
-{
-  "CN": "Wazuh",
-  "key": {
-    "algo": "rsa",
-    "size": 2048
-  },
-  "names": [
-  {
-    "C": "US",
-    "L": "San Francisco",
-    "O": "Wazuh",
-    "OU": "Wazuh Root CA"
-  }
- ]
-}
-EOF
-
-fi
-
-if [[ ! -e $fs/ca-config.json ]]; then
-$cfssl print-defaults config > ca-config.json
-fi
-
-gencert_rsa() {
-        name=$1
-        profile=$2
-cat << EOF | $cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=$profile -hostname="$name,127.0.0.1,localhost" - | $cfssljson -bare $name -
-{
-  "CN": "$i",
-  "key": {
-    "algo": "rsa",
-    "size": 2048
-  },
-  "names": [
-  {
-    "C": "US",
-    "L": "California",
-    "O": "Wazuh",
-    "OU": "Wazuh"
-  }
-  ],
-  "hosts": [
-    "$i",
-    "localhost"
-  ]
-}
-EOF
-openssl pkcs8 -topk8 -inform pem -in $name-key.pem -outform pem -nocrypt -out $name.key
-}
-
-gencert_ec() {
-    openssl ecparam -name secp256k1 -genkey -noout -out jwt-private.pem
-    openssl ec -in jwt-private.pem -pubout -out jwt-public.pem
-}
-
-hosts=(indexer dashboard mailpit)
-for i in "${hosts[@]}"; do
-        gencert_rsa $i www
-done
-
-users=(admin)
-for i in "${users[@]}"; do
-        gencert_rsa $i client
-done
-
-gencert_ec
-```
-
-</p>
-</details>
-
-1. Bring up the environment with `vagrant up`. Use the command provided in the console to start mailpit from within its VM. **mailpit** is configured to use TLS and access credentials (`admin:admin`). Use `ip addr` to check for the public IP address given to the VM and use that IP to access mailpit UI (e.g: `https://172.28.128.136:8025/`).
-2. Add the username and password for mailpit to the Wazuh Indexer keystore.
-    ```bash
-    echo "admin" | /usr/share/wazuh-indexer/bin/opensearch-keystore add opensearch.notifications.core.email.mailpit.username
-    echo "admin" | /usr/share/wazuh-indexer/bin/opensearch-keystore add opensearch.notifications.core.email.mailpit.password
-    chown wazuh-indexer:wazuh-indexer /etc/wazuh-indexer/opensearch.keystore
-    ```
-3. Ensure `mailpit` is accessible within the `server` VM (e.g `curl https://172.28.128.136:8025 -k -u admin:admin` should return HTML code). If not, add it to the list of known hosts in `/etc/hosts` (e.g `echo "172.28.128.136 mailpit mailpit" >> /etc/hosts`).
+# Reporting
```

Comment:
Is this file still required? If so, and will be updated later, we could add a `# TODO`